releases-beta/how-to-guides/resolving-a-cherry-pick-failure.md

```diff
@@ -0,0 +1,41 @@
+# Resolving a Cherry-Pick Failure
+
+This guide walks you through the steps to handle a failed cherry-pick during a release process in Aviator. Cherry-picking allows you to selectively integrate specific changes (commits) into a base release. If a conflict occurs, you can resolve it manually.
+
+Setup a repository, connect it to Aviator and [create a release project](./creating-a-release-project.md) for this guide.
+
+## Create a Base Release
+1. Add a new file to your repository. For example, `echo "base" > test.txt`
+2. Commit and push the changes to a different branch.
+3. Create a pull request to the main branch.
+4. Merge the PR into the main branch.
+![](../../.gitbook/assets/release-conflict-base.png)
+5. Cut a release: Create a new release based on this PR.
+![](../../.gitbook/assets/release-conflict-base-cut.png)
+
+## Raise Two more Pull Requests
```

Comment:
This can also be removed. WE should start this doc by explaining something like:

Cherry picking is a common workflow ...
But sometimes a cherry pick can fail due to a merge conflict. This doc will explain how to manage such scenario

Reply:
You can ignore this comment as well.